misc/build.func

```diff
@@ -491,6 +522,7 @@ install_script() {
   NEXTID=$(pvesh get /cluster/nextid)
   timezone=$(cat /etc/timezone)
   header_info
+  base_settings
```

Comment:
its wrong here? Need to replace with Row 529 ?

Reply:
no, essentially you load all the standard (base) settings (eg. 4GB HDD, 512Mb RAM, 1 CPU...) with the `base_settings` function, then calling `default_settings` allows to overide them with whatever is defined in the actual installation script (eg. 20Gb HDD, 4CPU...)